### ISSUE TITLE
Delay expressed relative to free flow traffic or "normal conditions". (Bugzilla Bug 475)

### ISSUE THREAD

**Issue by datexii** [DATEX-II-EU/DatexII#475]
This issue was created automatically with bugzilla2github.py
# Bugzilla Bug 475
Date: 2024-02-23T14:56:59+01:00
From: @iancornwellmottmac
To:   Bard de Vries &lt;<b.devries@u-trex.nl>&gt;

Last updated: 2024-02-23T14:58:10+01:00

**Comment by iancornwellmottmac** [comment]
done

**Comment by hbruch** [comment]
@iancornwell1 could you please add a reference to the change solving this issue? That would be helpful, thanks.

**Comment by iancornwell1** [comment]
I think we need a better agreed policy about issue closure and issue resolution. Sometimes we have closed when the change is committed - like in this case - but for the wider public user community that is not ideal. We should at most mark as resolved (not a built-in state but a label we have added), and close only when the change is available to users. And I agree that the resolution should give more information than I did above.
The change was committed to the DATEX II UML model in July 2024, for release to users in version 4.0, which is currently scheduled for release in late 2026 at the soonest.
